scripts/find-package-dependents.py

```diff
@@ -0,0 +1,228 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import logging
+import re
+import subprocess
+from collections import deque
+from pathlib import Path
+from typing import Dict, List, Set
+from urllib.parse import urlparse
+from hashlib import sha256
+
+EXTRACT_NAME_PATTERN = re.compile(r"^(?P<name>.*)-[^-]+-[^-]+$")
```

Comment:
we can probably drop this actually, it was needed before because goose couldn't handle the newline in `--qf '%{name}\n'`, but now that we have a static script we can just have it output things in the right way from the start.